modules/vue-pinia-shared-state/vuePinia/plugins/sharedState.ts

```diff
@@ -0,0 +1,13 @@
+import { PiniaSharedState } from 'pinia-shared-state'
+import type { App } from 'vue'
+
+export default (app: App, ctx: any) => {
```

Comment:
WHy isn't the pinia instance the first arguments instead of the app? Its plugin for pinia, not fo the vue app.